### DISCUSSION TITLE
Run status sensor for graph assets of specific asset group

### DISCUSSION THREAD

**Discussion by ignaski** [dagster-io/dagster#21234] [Q&A]
Hi, I am looking for a way on how to implement `run_status_sensor` which would monitor only assets from specific asset group. Assets are built using `graph_asset`. 
The problem that I am facing is that I can only get asset name from the run status context object. I could try to come up with some naming rules for the asset, but I feel this is not the correct approach.

**Comment by garethbrickman** [comment]
I don't believe there's a way at this time other than using tags to indicate the asset group
